webrtc.html

```diff
@@ -4096,8 +4096,15 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             "dom-rtptransceiver-mid"><code>mid</code></dfn>
             attribute is the <code>mid</code> negotatiated and present
             in the local and remote descriptions as defined in
-            <span data-jsep="initial-offers initial-answers">[[!JSEP]]</span>.  Before negotiation is complete,
-            the <code>mid</code> value may be null.  After rollbacks,
+            <span data-jsep="initial-offers
+	    initial-answers">[[!JSEP]]</span>.
+	    Before negotiation is complete,
+	    the <code>mid</code> value may be null.  If there is no
+	    MID value in the the remote SDP, a random value will be
+	    created for the <code>mid</code> as described in
```

Comment:
What happens if the offerer includes a MID in the offer, the answer comes back without a MID? According to above this should mean that a new (random) value is created, but should not the original MID be retained?